src/pages/blog/[slug]/opengraph-image.ts

```diff
@@ -0,0 +1,37 @@
+import { getCollection } from "astro:content";
+import { getSlug, loadFont, Template } from "../../../utils";
+import type { APIRoute } from "astro";
+import { GenerateImageResponse } from "../../../lib/astro-opengraph-image";
+
+export const prerender = true;
```

Comment:
if this is constant make it CAPITAL_CASE

Reply:
@Umeshmalik Astro needs it to be lowercase.

https://docs.astro.build/en/guides/server-side-rendering/#opting-in-to-pre-rendering-in-server-mode

Reply:
ok it is required for astro